setup.py

```diff
@@ -2,8 +2,8 @@
 import versioneer
 
 install_requires = ['menpo>=0.7,<0.8',
-                    'cyrasterize>=0.2.4,<0.3',
-                    'mayavi>=4.4.4']
+                    'cyrasterize>=0.3,<0.4',
+                    'mayavi>=4.5.0']
```

Comment:
I mean people will get it when they upgrade anyway right? Just wondering if there was a reason why we bumped it? But I'm pretty easy on it - I think it was all bug fixes anyway.